packages/wdio-allure-reporter/src/constants.js

```diff
@@ -13,7 +14,8 @@ const testStatuses = {
 const stepStatuses = {
     PASSED,
     FAILED,
-    BROKEN
+    BROKEN,
+    SKIPPED
```

Comment:
There is not `skipped` step status in Allure as I know

Reply:
So don't use it

Reply:
Using "pending" or "skipped" status marks the step as skipped with a grey icon within allure UI. I didn't want to assign a testStatuses.PENDING to a step, that is why I've added this SKIPPED status to stepStatuses.
From allure-js-common source code there is no official list of valid statuses for steps, but there is one for tests. That's rather weird..
https://github.com/allure-framework/allure-js-commons/blob/master/beans/step.js
https://github.com/allure-framework/allure-js-commons/blob/master/beans/test.js

Reply:
yeah got it. actually there is xsd scheme for xml here https://github.com/allure-framework/allure1/blob/master/allure-model/src/main/resources/allure.xsd#L139

and `skipped` is valid status, but it is deprecated and will be better use `pending` through. what do you think?

Reply:
They recommand using "canceled", so let's settle for that one :)